components/user_settings/notifications/email_notification_setting/email_notification_setting.jsx

```diff
@@ -54,11 +56,11 @@ export default class EmailNotificationSetting extends React.Component {
         });
     }
 
-    handleSubmit = () => {
+    handleSubmit = async () => {
```

Comment:
I don't see a need to make this `async` function.  I'd suggest not to change it.